main.c

```diff
@@ -57,6 +57,42 @@ int lenient_strcmp(char *a, char *b) {
 	}
 }
 
+#ifdef ENABLE_READYFD
+static void readyfd(void) {
+	int fd;
+	char *env = NULL, *endptr = NULL;
```

Comment:
Please declare those as needed.

Reply:
I assume you mean to not initialize them as NULL? I have done so in the most recent push. I am curious why that is a good practice.

Reply:
No, I mean declaring them when first used.

```c
int a, b;

… // Some code

a = f();
b = g();
```

vs.

```c
… // Some code

int a = f();
int b = g();
```

Reply:
done